cartoframes/assets/templates/error/basic.html.j2

```diff
@@ -0,0 +1,14 @@
+<div id="error-container" class="error">
+  <p>There is a <span class="errors" id="error-highlight">{% if type is defined %}{{ type }}{% endif %}</span>
```

Comment:
Isn't the updating of this purely done from JS? In other words, type, name or message are never going to be defined.

I'm worried such generic names eventually end up in the data for the template and we get some weird results

Reply:
Yes, you're right, I've to remove them.